homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,31 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int first = 1, second = 1, third;
+    
+    if(sequence == 0) {
+        return 0;
+    }
+    else if(sequence == 2 or sequence == 1) {
+        return first;
+    }
+
+    for(int i = 2; i < sequence; i++) {
+        third = first + second;
+        first = second;
+        second = third;
+    }
+    return third;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0) {
+        return 0;
+    }
+    else if(sequence == 1 or sequence == 2) {
+        return 1;
+    }
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

You avoid secodn if